app/assets/stylesheets/campaigner-facing/page-edit.scss

```diff
@@ -496,3 +496,27 @@ body.page-edit-body {
     opacity: 1;
   }
 }
+
+.call-tool {
+  .block {
+    display: block;
+  }
+
+  input[type="submit"] {
+    margin-left: 10px;
+  }
+
+  .error-msg {
+    color: #f8492e;
```

Comment:
why would call tool error messages be a different color than regular ones? we should only be using colors in colors.scss